src/shiv/bootstrap/__init__.py

```diff
@@ -228,7 +228,7 @@ def bootstrap():  # pragma: no cover
 
         if preamble_bin.suffix == ".py":
             runpy.run_path(
-                preamble_bin,
+                preamble_bin.as_posix(),
```

Comment:
I will be happy to ship this, but we I think should use `str(preamble_bin)` instead of the `.as_posix()` method, since `shiv` strives to be compatible on Windows

Reply:
That sounds perfectly reasonable. I can do an updated PR within the next day.

Reply:
I see the activity regarding the 1.0 release. I can make an update later today but not in the next 6 hours. Will it still make it into the 1.0 release?

Reply:
@mhucka I actually cherry-picked your commit into that 1.0.0 release and then in a subsequent commit (in that same PR) made the switch from my comment here. That way, you will still get contributor credit 👍

Reply:
Whoa! Thank you :-). And sorry I didn't look more carefully before commenting!

Reply:
No problem :)